website/docs/d/kms_secret.html.markdown

```diff
@@ -22,13 +22,14 @@ Please take care to secure your secret data outside of resource definitions.
 First, let's encrypt a password with KMS using the [AWS CLI
 tools](http://docs.aws.amazon.com/cli/latest/reference/kms/encrypt.html).  This
 requires you to have your AWS CLI setup correctly, and you would replace the
-key-id with your own.
+key-id with your own. Be sure you don't have an unexpected newline character when using a file as input.
```

Comment:
Could you add a small note about why you shouldn't have a newline there (as it may not be obvious for everyone)?